pkg/apis/apiresource/v1alpha1/common_types.go

```diff
@@ -62,6 +62,21 @@ func (cd *ColumnDefinitions) ImportFromCRDVersion(crdVersion *apiextensionsv1.Cu
 	return cd
 }
 
+func (cds *ColumnDefinitions) ToCustomResourceColumnDefinitions() []apiextensionsv1.CustomResourceColumnDefinition {
```

Comment:
we have the same code in the workload apiexport controller and in apiresource negotiation. Can you unify them to use this func?

Reply:
Done in amended commit 1045ce32c68848e5da84df98e746bc82f813b0ca